src/appComponents/tracks/TracksController.js

```diff
@@ -26,15 +28,15 @@ tracksModule.controller('TracksController',
         
 
         if (tc.Tracks.length === 0) {
-            ApiJsonFactory.getJson('tracks')
+            ApiJsonFactory.getJson("tracks")
                 .then(function (response) {
                     tc.Tracks = response.data.tracks;
                     $sessionStorage.tracks = tc.Tracks;
                 }, function (error) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 18 space characters but found 20.](https://www.codacy.com/app/triplez/open-event-webapp/file/2418082363/issues/source?bid=3233857&fileBranchId=3237740#l36)